### ISSUE TITLE
Installation fails on OS X 10.10.5

### ISSUE THREAD

**Issue by eproxus** [docker/toolbox#130]
The installation of Docker Toolbox 1.8.1b fails on OS X 10.10.1, with the following settings:

<img width="732" alt="screen shot 2015-08-27 at 11 05 52" src="https://cloud.githubusercontent.com/assets/112878/9516843/a319b9f0-4cab-11e5-8065-cc2b21cae3ed.png">

(Not that it says upgrade of all components even though Docker is completely removed from Homebrew and with the [uninstall script](https://github.com/docker/toolbox/blob/master/osx/uninstall.sh))

<img width="704" alt="screen shot 2015-08-27 at 11 03 03" src="https://cloud.githubusercontent.com/assets/112878/9516769/3d312f10-4cab-11e5-8b51-41bd6f0add76.png">

This is the installer log:

```
Aug 27 11:01:44 Mac.local Installer[21580]: LSExceptions [0x7f941350b070] loaded
Aug 27 11:01:44 Mac.local Installer[21580]: @(#)PROGRAM:Install  PROJECT:Install-924.1
Aug 27 11:01:44 Mac.local Installer[21580]: @(#)PROGRAM:Installer  PROJECT:Installer-815
Aug 27 11:01:44 Mac.local Installer[21580]: Hardware: iMac15,1 @ 4.00 GHz (x 8), 16384 MB RAM
Aug 27 11:01:44 Mac.local Installer[21580]: Running OS Build: Mac OS X 10.10.5 (14F27)
Aug 27 11:01:44 Mac.local Installer[21580]: Env: __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x0
Aug 27 11:01:44 Mac.local Installer[21580]: Env: TMPDIR=/var/folders/hl/5nvrk2bj40q1gvw6b301lx500000gn/T/
Aug 27 11:01:44 Mac.local Installer[21580]: Env: HOME=/Users/user
Aug 27 11:01:44 Mac.local Installer[21580]: Env: SHELL=/usr/local/bin/zsh
Aug 27 11:01:44 Mac.local Installer[21580]: Env: SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.w4es9Z93MC/Listeners
Aug 27 11:01:44 Mac.local Installer[21580]: Env: Apple_PubSub_Socket_Render=/private/tmp/com.apple.launchd.PtUout756n/Render
Aug 27 11:01:44 Mac.local Installer[21580]: Env: DISPLAY=/private/tmp/com.apple.launchd.KZciWpdi1u/org.macosforge.xquartz:0
Aug 27 11:01:44 Mac.local Installer[21580]: Env: PATH=/usr/bin:/bin:/usr/sbin:/sbin
Aug 27 11:01:44 Mac.local Installer[21580]: Env: LOGNAME=user
Aug 27 11:01:44 Mac.local Installer[21580]: Env: XPC_SERVICE_NAME=com.apple.xpc.launchd.oneshot.0x10000004.Installer
Aug 27 11:01:44 Mac.local Installer[21580]: Env: COMMAND_MODE=unix2003
Aug 27 11:01:44 Mac.local Installer[21580]: Env: USER=user
Aug 27 11:01:44 Mac.local Installer[21580]: Env: XPC_FLAGS=0x0
Aug 27 11:01:44 Mac.local Installer[21580]: Docker Toolbox %INSTALLER_VERSION% Installation Log
Aug 27 11:01:44 Mac.local Installer[21580]: Opened from: /Users/user/Downloads/DockerToolbox-1.8.1b.pkg
Aug 27 11:01:44 Mac.local Installer[21580]: Product archive /Users/user/Downloads/DockerToolbox-1.8.1b.pkg trustLevel=202
Aug 27 11:01:48 Mac.local Installer[21580]: JS: Hardware architecture detected: x86_64
Aug 27 11:01:48 Mac.local Installer[21580]: JS: OS version detected: 10.10.5
Aug 27 11:01:48 Mac.local Installer[21580]: JS: 5.0.0
Aug 27 11:01:48 Mac.local Installer[21580]: JS: system.run /bin/sh .. returned: 1 result=true
Aug 27 11:01:48 Mac.local Installer[21580]: JS: result:true
Aug 27 11:01:49 Mac.local Installer[21580]: InstallerStatusNotifications plugin loaded
Aug 27 11:01:55 Mac.local Installer[21580]: LSExceptions [0x7f941350b070] unloaded
Aug 27 11:01:59 Mac.local runner[21612]: Administrator authorization granted.
Aug 27 11:01:59 Mac.local Installer[21580]: ================================================================================
Aug 27 11:01:59 Mac.local Installer[21580]: User picked Custom Install
Aug 27 11:01:59 Mac.local Installer[21580]: Choices selected for installation:
Aug 27 11:01:59 Mac.local Installer[21580]:     Upgrade: "Docker Toolbox"
Aug 27 11:01:59 Mac.local Installer[21580]:     Upgrade: "Docker Client 1.8.1"
Aug 27 11:01:59 Mac.local Installer[21580]:         DockerToolbox-1.8.1b.pkg#docker.pkg : io.docker.pkg.docker : 1.8.1
Aug 27 11:01:59 Mac.local Installer[21580]:     Upgrade: "Docker Machine 0.4.1"
Aug 27 11:01:59 Mac.local Installer[21580]:         DockerToolbox-1.8.1b.pkg#dockermachine.pkg : io.docker.pkg.dockermachine : 0.4.1
Aug 27 11:01:59 Mac.local Installer[21580]:     Upgrade: "Docker Compose 1.4.0"
Aug 27 11:01:59 Mac.local Installer[21580]:         DockerToolbox-1.8.1b.pkg#dockercompose.pkg : io.docker.pkg.dockercompose : 1.4.0
Aug 27 11:01:59 Mac.local Installer[21580]:     Upgrade: "Docker Quickstart Terminal app"
Aug 27 11:01:59 Mac.local Installer[21580]:         DockerToolbox-1.8.1b.pkg#dockerquickstartterminalapp.pkg : io.docker.pkg.dockerquickstartterminalapp : 1.8.1b
Aug 27 11:01:59 Mac.local Installer[21580]:     Upgrade: "Kitematic 0.8.3"
Aug 27 11:01:59 Mac.local Installer[21580]:         DockerToolbox-1.8.1b.pkg#kitematicapp.pkg : io.docker.pkg.kitematicapp : 0.8.3
Aug 27 11:01:59 Mac.local Installer[21580]:         DockerToolbox-1.8.1b.pkg#boot2dockeriso.pkg : io.boot2dockeriso.pkg.boot2dockeriso : 1.8.1
Aug 27 11:01:59 Mac.local Installer[21580]: ================================================================================
Aug 27 11:01:59 Mac.local Installer[21580]: It took 0.00 seconds to summarize the package selections.
Aug 27 11:01:59 Mac.local Installer[21580]: -[IFDInstallController(Private) _buildInstallPlan]: location = file://localhost
Aug 27 11:01:59 Mac.local Installer[21580]: -[IFDInstallController(Private) _buildInstallPlan]: file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#docker.pkg
Aug 27 11:01:59 Mac.local Installer[21580]: -[IFDInstallController(Private) _buildInstallPlan]: file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#dockermachine.pkg
Aug 27 11:01:59 Mac.local Installer[21580]: -[IFDInstallController(Private) _buildInstallPlan]: file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#dockercompose.pkg
Aug 27 11:01:59 Mac.local Installer[21580]: -[IFDInstallController(Private) _buildInstallPlan]: file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#dockerquickstartterminalapp.pkg
Aug 27 11:01:59 Mac.local Installer[21580]: -[IFDInstallController(Private) _buildInstallPlan]: file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#kitematicapp.pkg
Aug 27 11:01:59 Mac.local Installer[21580]: -[IFDInstallController(Private) _buildInstallPlan]: file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#boot2dockeriso.pkg
Aug 27 11:01:59 Mac.local Installer[21580]: Set authorization level to root for session
Aug 27 11:01:59 Mac.local Installer[21580]: Will use PK session
Aug 27 11:01:59 Mac.local Installer[21580]: Starting installation:
Aug 27 11:01:59 Mac.local Installer[21580]: Configuring volume "Macintosh HD"
Aug 27 11:01:59 Mac.local Installer[21580]: Preparing disk for local booted install.
Aug 27 11:01:59 Mac.local Installer[21580]: Free space on "Macintosh HD": 678.95 GB (678951821312 bytes).
Aug 27 11:01:59 Mac.local Installer[21580]: Create temporary directory "/var/folders/hl/5nvrk2bj40q1gvw6b301lx500000gn/T//Install.21580ik6Alh"
Aug 27 11:01:59 Mac.local Installer[21580]: IFPKInstallElement (6 packages)
Aug 27 11:01:59 Mac.local Installer[21580]: Using authorization level of root for IFPKInstallElement
Aug 27 11:01:59 Mac.local installd[21298]: PackageKit: Adding client PKInstallDaemonClient pid=21580, uid=501 (/System/Library/CoreServices/Installer.app/Contents/MacOS/Installer)
Aug 27 11:01:59 Mac.local Installer[21580]: PackageKit: Enqueuing install with framework-specified quality of service (utility)
Aug 27 11:01:59 Mac.local installd[21298]: PackageKit: ----- Begin install -----
Aug 27 11:01:59 Mac.local installd[21298]: PackageKit: request=PKInstallRequest <6 packages, destination=/>
Aug 27 11:01:59 Mac.local installd[21298]: PackageKit: packages=(
        "PKLeopardPackage <file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#docker.pkg>",
        "PKLeopardPackage <file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#dockermachine.pkg>",
        "PKLeopardPackage <file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#dockercompose.pkg>",
        "PKLeopardPackage <file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#dockerquickstartterminalapp.pkg>",
        "PKLeopardPackage <file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#kitematicapp.pkg>",
        "PKLeopardPackage <file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#boot2dockeriso.pkg>"
    )
Aug 27 11:02:00 Mac.local installd[21298]: PackageKit: Will do receipt-based obsoleting for package identifier io.docker.pkg.docker (prefix path=)
Aug 27 11:02:00 Mac.local installd[21298]: PackageKit: Will do receipt-based obsoleting for package identifier io.docker.pkg.dockermachine (prefix path=)
Aug 27 11:02:00 Mac.local installd[21298]: PackageKit: Will do receipt-based obsoleting for package identifier io.docker.pkg.dockercompose (prefix path=)
Aug 27 11:02:00 Mac.local installd[21298]: PackageKit: Will do receipt-based obsoleting for package identifier io.docker.pkg.dockerquickstartterminalapp (prefix path=Applications/Docker/)
Aug 27 11:02:00 Mac.local installd[21298]: PackageKit: Will do receipt-based obsoleting for package identifier io.docker.pkg.kitematicapp (prefix path=Applications/Docker/)
Aug 27 11:02:00 Mac.local installd[21298]: PackageKit: Will do receipt-based obsoleting for package identifier io.boot2dockeriso.pkg.boot2dockeriso (prefix path=usr/local/share/boot2docker)
Aug 27 11:02:00 Mac.local installd[21298]: PackageKit: Extracting file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#docker.pkg (destination=/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/PKInstallSandboxManager/4436DD55-1E10-4BD7-B826-36E3BDFD7D40.activeSandbox/Root, uid=0)
Aug 27 11:02:00 Mac.local installd[21298]: PackageKit: Extracting file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#dockermachine.pkg (destination=/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/PKInstallSandboxManager/4436DD55-1E10-4BD7-B826-36E3BDFD7D40.activeSandbox/Root, uid=0)
Aug 27 11:02:00 Mac.local installd[21298]: PackageKit: Extracting file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#dockercompose.pkg (destination=/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/PKInstallSandboxManager/4436DD55-1E10-4BD7-B826-36E3BDFD7D40.activeSandbox/Root, uid=0)
Aug 27 11:02:00 Mac.local installd[21298]: PackageKit: Extracting file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#dockerquickstartterminalapp.pkg (destination=/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/PKInstallSandboxManager/4436DD55-1E10-4BD7-B826-36E3BDFD7D40.activeSandbox/Root/Applications/Docker, uid=0)
Aug 27 11:02:00 Mac.local installd[21298]: PackageKit: Extracting file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#kitematicapp.pkg (destination=/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/PKInstallSandboxManager/4436DD55-1E10-4BD7-B826-36E3BDFD7D40.activeSandbox/Root/Applications/Docker, uid=0)
Aug 27 11:02:01 Mac.local installd[21298]: PackageKit: Extracting file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#boot2dockeriso.pkg (destination=/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/PKInstallSandboxManager/4436DD55-1E10-4BD7-B826-36E3BDFD7D40.activeSandbox/Root/usr/local/share/boot2docker, uid=0)
Aug 27 11:02:01 Mac.local installd[21298]: PackageKit: prevent user idle system sleep
Aug 27 11:02:01 Mac.local installd[21298]: PackageKit: suspending backupd
Aug 27 11:02:01 Mac.local installd[21298]: PackageKit: Using trashcan path /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/PKInstallSandboxTrash/4436DD55-1E10-4BD7-B826-36E3BDFD7D40.sandboxTrash for sandbox /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/PKInstallSandboxManager/4436DD55-1E10-4BD7-B826-36E3BDFD7D40.activeSandbox
Aug 27 11:02:01 Mac.local installd[21298]: PackageKit: Shoving /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/PKInstallSandboxManager/4436DD55-1E10-4BD7-B826-36E3BDFD7D40.activeSandbox/Root (2 items) to /
Aug 27 11:02:01 Mac.local install_monitor[21630]: Temporarily excluding: /Applications, /Library, /System, /bin, /private, /sbin, /usr
Aug 27 11:02:01 Mac.local installd[21298]: PackageKit: Executing script "./postinstall" in /private/tmp/PKInstallSandbox.yikYrK/Scripts/io.boot2dockeriso.pkg.boot2dockeriso.a2iQMx
Aug 27 11:02:02 Mac.local installd[21298]: ./postinstall:    --virtualbox-import-boot2docker-vm                                             The name of a Boot2Docker VM to import
Aug 27 11:02:03 Mac.local installd[21298]: ./postinstall:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
Aug 27 11:02:03 Mac.local installd[21298]: ./postinstall:                                  Dload  Upload   Total   Spent    Left  Speed
Aug 27 11:02:04 Mac.local installd[21298]: ./postinstall: 
      0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (7) Couldn't connect to server
Aug 27 11:02:04 Mac.local install_monitor[21630]: Re-included: /Applications, /Library, /System, /bin, /private, /sbin, /usr
Aug 27 11:02:05 Mac.local installd[21298]: PackageKit: releasing backupd
Aug 27 11:02:05 Mac.local installd[21298]: PackageKit: allow user idle system sleep
Aug 27 11:02:05 Mac.local installd[21298]: PackageKit: Install Failed: Error Domain=PKInstallErrorDomain Code=112 "An error occurred while running scripts from the package “DockerToolbox-1.8.1b.pkg”." UserInfo=0x7fd93d00df70 {NSFilePath=./postinstall, NSURL=file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#boot2dockeriso.pkg, PKInstallPackageIdentifier=io.boot2dockeriso.pkg.boot2dockeriso, NSLocalizedDescription=An error occurred while running scripts from the package “DockerToolbox-1.8.1b.pkg”.} {
        NSFilePath = "./postinstall";
        NSLocalizedDescription = "An error occurred while running scripts from the package \U201cDockerToolbox-1.8.1b.pkg\U201d.";
        NSURL = "file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#boot2dockeriso.pkg";
        PKInstallPackageIdentifier = "io.boot2dockeriso.pkg.boot2dockeriso";
    }
Aug 27 11:02:05 Mac.local installd[21298]: PackageKit: Running idle tasks
Aug 27 11:02:05 Mac.local Installer[21580]: install:didFailWithError:Error Domain=PKInstallErrorDomain Code=112 "An error occurred while running scripts from the package “DockerToolbox-1.8.1b.pkg”." UserInfo=0x7f9416001850 {NSFilePath=./postinstall, NSURL=file://localhost/Users/user/Downloads/DockerToolbox-1.8.1b.pkg#boot2dockeriso.pkg, PKInstallPackageIdentifier=io.boot2dockeriso.pkg.boot2dockeriso, NSLocalizedDescription=An error occurred while running scripts from the package “DockerToolbox-1.8.1b.pkg”.}
Aug 27 11:02:05 Mac.local installd[21298]: PackageKit: Removing client PKInstallDaemonClient pid=21580, uid=501 (/System/Library/CoreServices/Installer.app/Contents/MacOS/Installer)
Aug 27 11:02:05 Mac.local installd[21298]: PackageKit: Done with sandbox removals
Aug 27 11:02:05 Mac.local installd[21298]: PackageKit: Sandbox /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/PKInstallSandboxManager/9D1AA9A5-AC5B-45F8-809A-76D51A3BDDF0.sandbox: relevance 265
Aug 27 11:02:05 Mac.local Installer[21580]: Install failed: The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.
Aug 27 11:02:05 Mac.local Installer[21580]: IFDInstallController 1346B0F0 state = 8
Aug 27 11:02:05 Mac.local Installer[21580]: Displaying 'Install Failed' UI.
Aug 27 11:02:05 Mac.local Installer[21580]: 'Install Failed' UI displayed message:'The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.'.

```

**Comment by jeffdm** [comment]
This should be fixed in 1.8.1c. Thanks again!

**Comment by eproxus** [comment]
Thanks a lot!
